stacks-core/src/hash/sha256.rs

```diff
@@ -0,0 +1,135 @@
+use ruint::aliases::U256;
```

Comment:
We already have a `Uint256` struct we use pervasively throughout the code. Please use that instead.

Reply:
I've now done that.